docs/zh-CN/community/developer-guide/docker-dev.md

```diff
@@ -29,84 +29,27 @@ under the License.
 
 ## 相关详细文档导航
 
-- [使用 Docker 开发镜像编译](/docs/install/source-install/compilation)
-- [部署](/docs/install/install-deploy)
-- [VSCode Be 开发调试](./be-vscode-dev)
+- [使用 Docker 开发镜像编译](../../docs/install/source-install/compilation.md)
+- [部署](../../docs/install/install-deploy.md)
+- [VSCode Be 开发调试](./be-vscode-dev.md)
```

Comment:
I'm not sure to add `.md` and `../../`is right.
PLTA, @hf200012

Reply:
It's right